roles/confluent.kafka_connect/tasks/rbac.yml

```diff
@@ -21,46 +21,46 @@
   shell: |
     curl -X POST {{mds_http_protocol}}://{{ groups['kafka_broker'][0] }}:{{mds_port}}/security/1.0/principals/User:{{item}}/roles/SystemAdmin \
```

Comment:
How does this interface with the URI work?  Is the plan to overwrite this with URI?  I think this will be a merge conflict.

Reply:
yep, merge conflicts are coming, I plan to merge the uri change first and then resolve this pr